docs/pages/_error.tsx

```diff
@@ -1,17 +1,20 @@
 import * as Sentry from "@sentry/nextjs";
```

Comment:
These file's changes come from https://docs.sentry.io/platforms/javascript/guides/nextjs/manual-setup/#react-render-errors-in-pages-router

We had the JS version, now it's the TS version.